apps/extension/src/routes/page/onboarding/routes.tsx

```diff
@@ -14,21 +16,26 @@ export const onboardingRoutes = [
   {
     path: PagePath.GENERATE_SEED_PHRASE,
     element: <GenerateSeedPhrase />,
+    loader: onboardingIndexLoader,
```

Comment:
Did you see my last comment on the `onboardingIndexLoader`?

Reply:
Think you should delete it

Reply:
I don't understand why we have to remove noPasswordIndexLoader and onboardingIndexLoader! a user without them can enter links and be thrown to any link he shouldn't see! for example, a logged user can go to the login page

Reply:
I understood that you used keplr as a basis and you can add a few seed phrases to them

Reply:
if we want to use the keplr approach then we should use tab mode only for creating a new account! all other functionality should be moved to popup! because now I made an example of metamask

Reply:
How is it possible a user ends up on these pages they shouldn't be on? Unless it's a security issue, I don't think we need to guard for events that are effectively not possible.

Whether we use tabs or popup should entirely be on the best UX possible for our users.

Reply:
we need to decide how our extensions should work, we are currently looking at different prototypes (Metamask and keplr)